server/app/views/components/FieldWithLabel.java

```diff
@@ -276,6 +278,11 @@ public FieldWithLabel setDisabled(boolean disabled) {
     return this;
   }
 
+  public FieldWithLabel setRequired(boolean required) {
```

Comment:
suggestion: All the usage of this is passing in true. We can probably remove the boolean altogether.
```suggestion
  public FieldWithLabel isRequired() {
```